pypeit/spectrographs/p200_dbsp.py

```diff
@@ -0,0 +1,422 @@
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'p200_dbsp_blue_600_4000.fits'
+        else:
+            msgs.error("Your grating " + disp + ' needs a template spectrum for the blue arm of DBSP.')
+        return par
+
+
+class P200DBSPRedSpectrograph(P200DBSPSpectrograph):
+    """
+    Child to handle P200/DBSPr red specific code
+    """
+    def __init__(self):
+        # Get it started
+        super(P200DBSPRedSpectrograph, self).__init__()
+        self.spectrograph = 'p200_dbsp_red'
+        self.camera = 'DBSPr'
+    
+    def compound_meta(self, headarr: List[fits.Header], meta_key: str):
+        retval = super(P200DBSPRedSpectrograph, self).compound_meta(headarr, meta_key)
+        if retval is not None:
+            return retval
+        if meta_key == 'binning':
+            binspec, binspatial = headarr[0]['CCDSUM'].split(' ')
+            return parse.binning2string(binspec, binspatial)
+        else:
+            msgs.error("Not ready for this compound meta")
+
+    def get_detector_par(self, hdu: fits.HDUList, det: int):
+        """
+        Return a DectectorContainer for the current image
+
+        Args:
+            hdu (`astropy.io.fits.HDUList`):
+                HDUList of the image of interest.
+                Ought to be the raw file, or else..
+            det (int):
+
+        Returns:
+            :class:`pypeit.images.detector_container.DetectorContainer`:
+
+        """
+        # Binning
+        binning = self.get_meta_value(self.get_headarr(hdu), 'binning')  # Could this be detector dependent??
+
+        # Detector 1
+        detector_dict = dict(
+            binning         = binning,
+            det             = 1,
+            dataext         = 0,
+            specaxis        = 1,
+            specflip        = False,
+            spatflip        = False, # check
+            platescale      = 0.293,
+            darkcurr        = 0.0,
+            saturation      = 45000.,
+            nonlinear       = 40./45.,
+            mincounts       = -1e10, # check
+            numamplifiers   = 1,
+            gain            = np.atleast_1d(2.8),
+            ronoise         = np.atleast_1d(8.5)
+        )
+
+        header = hdu[0].header
+
+        datasec = header['DSEC1']
+        oscansec = header['BSEC1']
+
+        detector_dict['datasec'] = np.atleast_1d(flip_fits_slice(datasec))
+        detector_dict['oscansec'] = np.atleast_1d(flip_fits_slice(oscansec))
+
+        return detector_container.DetectorContainer(**detector_dict)
+
+    def default_pypeit_par(self):
+        """
+        Set default parameters for P200 DBSPr reductions.
+        """
+        par = pypeitpar.PypeItPar()
+        par['rdx']['spectrograph'] = 'p200_dbsp_red'
+
+        # Ignore PCA
+        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+        #par['calibrations']['slitedges']['edge_thresh'] = 'nearest'
+
+        # Turn off the overscan
+        #for ftype in par['calibrations'].keys():
```

Comment:
perhaps these commented lines can be removed? The overscan is set to True below, so maybe the above comment needs updating?